light_test.go

```diff
@@ -341,6 +371,7 @@ func TestSetLightAlert(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	assert.True(t, light.IsOn())
```

Comment:
This line is why tests fails. It doesn't receive expected result. Can you have a look?